applications/gafaelfawr/templates/ingress.yaml

```diff
@@ -47,3 +47,48 @@ spec:
                 port:
                   number: 8080
           {{- end }}
+    {{- $context := . }}
+    {{- with $context.Values.config.additionalFQDNs }}
+    {{- range . }}
+    - host: {{ . | quote }}
+      http:
+        paths:
+          - path: "/auth"
+            pathType: Prefix
+            backend:
+              service:
+                name: {{ template "gafaelfawr.fullname" $context }}
+                port:
+                  number: 8080
+          - path: "/login"
+            pathType: Exact
+            backend:
+              service:
+                name: {{ template "gafaelfawr.fullname" $context }}
+                port:
+                  number: 8080
+          - path: "/logout"
+            pathType: Exact
+            backend:
+              service:
+                name: {{ template "gafaelfawr.fullname" $context }}
+                port:
+                  number: 8080
+          {{- if $context.Values.config.oidcServer.enabled }}
+          - path: "/.well-known/jwks.json"
+            pathType: Exact
+            backend:
+              service:
+                name: {{ template "gafaelfawr.fullname" $context }}
+                port:
+                  number: 8080
+          - path: "/.well-known/openid-configuration"
+            pathType: Exact
+            backend:
+              service:
+                name: {{ template "gafaelfawr.fullname" $context }}
+                port:
+                  number: 8080
+          {{- end }}
+    {{- end }}
```

Comment:
None of this stuff is going to work for right now, so let's only add `/auth` for additional hosts. (Once we have proper multiple domain support, we won't be exposing those URLs in every hostname anyway.)